src/App.module.scss

```diff
@@ -0,0 +1,37 @@
+.quizRecap {
+  background-color: white;
+  border-radius: 4px;
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-top: 16px;
+  padding-bottom: 16px;
+
+  .h2 {
```

Comment:
With CSS modules there is no need to nest all these classes, right? Or could an`.h2` class definition in the main application CSS still bled into the component?

Reply:
CSS modules scope styles locally to the component. I faced an issue that CSS cascaded through nested components. The parent component applied style to `p` and the `p` child component was affected. But I'll double check.

Reply:
![image](https://github.com/christophblessing/quiz-recap/assets/22977799/4d42ce90-8e44-466c-b4a7-503daf63b662)

Here is an example. I applied a `background-color` to every `<p>` in the `App.module.scss`- as you can see all nested `<p>` are also with this colour. They are part of the `Question` and `Answers` component.

Reply:
> this might be better for the case where you need to override a style from the outside

Should we encourage that? Isn't it better to have a higher specificity so that styles are not accidentally overridden? Styles we want to override should be exposed as variables (such as colours).

Reply:
No, actually not. Fine as it is for me 👍